nagios.yml

```diff
@@ -136,3 +138,7 @@ parameters:
   name: "NAGIOS_PASSWORD"
   generate: "expression"
   from: "[a-zA-Z0-9]{10}"
+-
+  description: "Mbaas router dns"
+  name: "MBAAS_ROUTER_DNS"
+  value: "localhost"
```

Comment:
Would 'fh-mbaas-service' be a better default?
This would mean it wouldn't have to be set by the user in majority of use cases.